docs/tutorials/single_screen.rst

```diff
@@ -311,3 +323,88 @@ Now, show the dynamic spectrum.
 
     plt.show()
 
+Create the conjugate spectrum and the secondary spectrum
+========================================================
+
+The conjugate spectrum refers to the Fourier transform of the dynamic spectrum.
+
+.. math::
+
+    \tilde{I}(\tau, f_\mathrm{D}) = \mathcal{F}[ I(f, t) ]
+
+Here, the conjugate spectrum is created and normalized to its zero-frequency
+component, which is equivalent to normalizing to the mean of the dynamic
+spectrum. Afterwards, the zero-frequency component is shifted to the centre of
+the spectrum.
+
+.. jupyter-execute::
+
+    conspec = np.fft.fft2(dynspec)
+    conspec /= conspec[0, 0]
+    conspec = np.fft.fftshift(conspec)
+
+The conjugate variables, the relative geometric delay :math:`\tau` and the
+differential Doppler shift :math:`f_\mathrm{D}`, also need to be created and
+shifted.
+
+.. jupyter-execute::
+
+    tau = np.fft.fftfreq(dynspec.shape[1], f[0,1] - f[0,0]).to(u.us)
+    fd = np.fft.fftfreq(dynspec.shape[0], t[1,0] - t[0,0]).to(u.mHz)
+
+    tau = np.fft.fftshift(tau)
+    fd = np.fft.fftshift(fd)
+
+.. note::
```

Comment:
On second thought, I think we should just change this class's name and then document both it and `DynamicSpectrum`. For now, I've removed the note...